Frontend-Angular/data-polish-frontend/src/assets/Dataprofile_config.json

```diff
@@ -11,29 +11,89 @@
+        "WHAT":"Displays the count of Records in the dataset.",
+        "WHY":"In order to understand the Dataset size"
         
         
     },
-    "NUMBEROFFIELDS":{
-        "WHAT":"Displays the count of fields in the dataset.",
-        "WHY":""
+    "bubble-chart":{
+        "WHAT":"Visualises the categorical features within a column as bubbles.",
+        "WHY":"Use to identify categorical distribution. Large bubbles may indicate asimetric data points needing normalisation."
         
         
     },
-    "NUMBEROFFIELDS":{
-        "WHAT":"Displays the count of fields in the dataset.",
-        "WHY":""
+    "cardinality":{
+        "WHAT":"Shows the uniqueness of data by representing the number of distinct values in a column.",
+        "WHY":"Helpful for understanding when to collapse rare categories or when binning might be necessary for numeric to categorical conversions."
         
         
     },
-    "NUMBEROFFIELDS":{
-        "WHAT":"Displays the count of fields in the dataset.",
-        "WHY":""
+    "correlation-scatter-plot":{
+        "WHAT":"Illustrates the relationship between two variables and their correlation coefficient.",
+        "WHY":"Use to determine if fields are redundant (high correlation) or if PCA might be useful for dimensionality reduction."
+        
+ 
+    },
+    "data-quality-metric":{
+        "WHAT":"Evaluates data quality across four dimensions: completeness, uniqueness, readability, and consistency, providing a comprehensive overview of data health.",
+        "WHY":"Used as a Estimate to determine the readiness of the dataset before machine learning."
+        
+ 
+    },
+    "data-types-chart":{
+        "WHAT":"Categorizes data into different types (numeric, categorical, etc.).",
+        "WHY":"Allows for a better fundemental understanding of data type distribution"
+        
+ 
+    },
+    "duplicate-chart":{
+        "WHAT":"Visualizes the frequency of duplicate records in a dataset.",
+        "WHY":"shows duplicate records to be removed or merged."
         
+ 
+    },
+    "histogram":{
+        "WHAT":"Represents the distribution of numerical data.",
+        "WHY":"Analyze the distribution of values to inform normalization."
+        
+ 
+    },
+    "missing-values-chart":{
+        "WHAT":"Highlights the number of missing values within a column in a dataset.",
+        "WHY":"Directs where to fill missing values or to decide if a column with too many missing values should be removed."
+        
+ 
+    },
+    "outliers-scatter-plot":{
+        "WHAT":"Identifies outlier data points in a dataset.",
+        "WHY":"Pinpoints outliers for removal or further examination."
+        
+ 
+    },
+    "completeness-metric":{
+        "WHAT":"Measures the proportion of non-missing entries in the dataset to assess data completeness.",
+        "HOW TO IMPOROVE":"to improve the score, look into the 'Fill Missing Values' function."
         
+ 
+    },
+    "uniqueness-metric":{
```

Comment:
Would mention curse of dimensionality and why that impacts but overall like this